tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - include: nginx-official-repo.yml
   when: nginx_official_repo == True
+- include: nginx-mainline-repo.yml
+  when: nginx_mainline_repo == True
```

Comment:
Why is it important? It is two separated, the mainline not depend from the use repo.